docs/lightning/release-notes.md

```diff
@@ -8,7 +8,7 @@ Important: when upgrading the CTI Adapter, please make sure that the Salesforce
 ## 5.15 July 2021
 
 - **Feature: Guided Setup** The Guided Setup feature helps make the setup process easier. See [Guided Setup](/docs/lightning/installation/02-guided-setup) for more details.
-- **Feature: Chat Widget Integration for SalesForce Experience Cloud** Added VisualForce Page component that allows you to add Amazon Connect Chat Widget in Your Experience Cloud Site.
+- **Feature: Chat Widget Integration for SalesForce Service and Sales Cloud** Added VisualForce Page component that allows you to add Amazon Connect Chat Widget in your Salesforce communities website.
```

Comment:
should this be in classic release notes as well?

Reply:
my bad forgot that!